Sources/Depo/Commands/Example.swift

```diff
@@ -0,0 +1,20 @@
+//
+// Copyright © 2021 Rosberry. All rights reserved.
+//
+
+import Foundation
+import DepoCore
+import ArgumentParser
+
+final class Example: ParsableCommand {
+    static let configuration: CommandConfiguration = .init(commandName: "example",
+                                                           abstract: "prints example of Depofile")
+
+    private lazy var shell: Shell = Shell()
+
+    func run() throws {
+        let exampleURL = "https://raw.githubusercontent.com/rosberry/depo/master/DepofileExample.yaml"
```

Comment:
It seems strange to fetch an example like that.

Imagine the following case: 
You've installed Depo, you've tried it, everything works, all is fine. Time passes. 
There were some changes in the upstream, maybe even breaking ones, and Depofile parameters have changed.
But you didn't update yet and you want to check an example to setup Depo for the new project. You'll get an example which may be completely incompatible with the version you're using, since it will be an example for the latest version.

Reply:
Nice point. I've fixed this by getting DepofileExample.yaml not from master, but from tag, since depo version and tag should be the same